aml-development/ozp-center
# Org steward modal title

To test:
Go to Submit Listing page
Click the 'View list of organization stewards' link.
Verify the modal has the title bar with the X.

@mleeBoeing, @flesher1, @rkenyon1969, @lsemesky
